src/test/java/com/uwetrottmann/tmdb/services/MoviesServiceTest.java

```diff
@@ -199,6 +201,16 @@ public void test_videos() {
     }
 
     @Test
+    public void test_translations() {
+        Translations translations = getManager().moviesService().translations(TestData.MOVIE_ID, null);
+        assertThat(translations).isNotNull();
+        assertThat(translations.id).isEqualTo(TestData.MOVIE_ID);
+        assertThat(translations.translations.get(0).name).isNotNull();
```

Comment:
Probably better to for each loop through all `Translation` objects to make sure all of them are valid.